examples/send_self.rs

```diff
@@ -19,7 +19,8 @@ fn main() {
 
     let handle = bot.new_cmd("/send_self").and_then(|(bot, msg)| {
         bot.document(msg.chat.id)
-            .file("examples/send_self.rs")
+            .try_file("examples/send_self.rs")
```

Comment:
Perhaps bot.document(msg.chat.id).file("").try_send() is another way, with try_send propagating a possible error the stream chain

Reply:
`.send()` should fail now if the file was not properly attached. I created a `NoFile` error which gets passed through the send logic if there should have been a file attached but it was unable to be converted into a file.